src/_guides/language/language-tour.md

```diff
@@ -3033,6 +3033,51 @@ class Point {
 }
 ```
 
+#### Operators {#_operators}
+
+Operators are instance methods with special names.
+Dart allows you to define operators with the following names:
+
+`<`  | `+`  | `|`  | `[]`
+`>`  | `/`  | `^`  | `[]=`
+`<=` | `~/` | `&`  | `~`
+`>=` | `*`  | `<<` | `==`
+`–`  | `%`  | `>>`
+{:.table}
+
+{{site.alert.note}}
+  You may have noticed that some [operators](#operators), like `!=`, are not in
+  the list of names. That's because they're just syntactic sugar. For example,
+  the expression `e1 != e2` is syntactic sugar for `!(e1 == e2)`.
+{{site.alert.end}}
```

Comment:
Maybe worth mentioning that `??`, `&&` and ||` are not in the list either because they are not, and cannot be, methods because of their short-circuiting nature.

(And `!` is not there for ... probably historical reasons. No real reason I can think of.)

Reply:
Thanks for the feedback.

@kwalrath - thoughts?

AFAIK, in the language tour we generally try to avoid (too many) low-level technical details (Kathy can correct me if I'm wrong). So ...

> Maybe worth mentioning that ??, && and || are not in the list either because they are not, and cannot be, methods because of their short-circuiting nature.

... would make a great commentary in the language spec (AI for @eernstg? 😄)

> And ! is not there for ... probably historical reasons. No real reason I can think of.

Interesting! (Also possibly a good commentary for the DLS?)

Reply:
Agreed that we don't want to show these kinds of details in the tour. (It'd be fine to note them or this comment in `{% comment %}` sections, so future writers know why they're omitted.)

Reply:
Good idea, I'll add those as comments.